include/cute/arch/copy_sm80.hpp

```diff
@@ -57,7 +57,7 @@ struct SM80_CP_ASYNC_CACHEALWAYS
   copy(TS const& gmem_src,
        TD      & smem_dst)
   {
-#if defined(CUTE_ARCH_CP_ASYNC_SM80_ENABLED) && defined(ENABLE_NVPTX)
+#if defined(CUTE_ARCH_CP_ASYNC_SM80_ENABLED)
```

Comment:
No need to check for `ENABLE_NVPTX`. `CUTE_ARCH_CP_ASYNC_SM80_ENABLED` depends on `__SYCL_CUDA_ARCH__` being enabled